src/Faker/Provider/pt_BR/Person.php

```diff
@@ -99,4 +99,56 @@ public static function suffix()
     {
         return static::randomElement(static::$suffix);
     }
+
+
+    /**
+     * A random CPF number.
+     * @param bool $formatted If the number should have dots/dashes or not.
+     * @return string
+     */
+    public function cpf($formatted = true)
+    {
+        $n = $this->generator->numerify('#########');
+        $n .= $this->verifierDigit($n);
```

Comment:
If you always double it, why don't the `verifierDigit()` return it twice instead?